crates/settings/src/settings.rs

```diff
@@ -66,6 +66,8 @@ pub fn init(cx: &mut AppContext) {
         .set_default_settings(&default_settings(), cx)
         .unwrap();
     cx.set_global(settings);
+
+    ImageFileSizeUnitType::register(cx);
```

Comment:
This shouldn't be registered here. This should be in an `init()` function in the `image_info` file.

Reply:
So for this scenario... i tried having an exact `init` function in `ImageItem` But I kept on running into an error. This one specifically. 

```shell
Thread "main" panicked with "unregistered setting type image_viewer::image_info::ImageFileSizeUnitType" at /oss/zed/crates/settings/src/settings_store.rs:341:32
   0: zed::reliability::init_panic_hook::{{closure}}
```

So to avoid it, I ended up using `call_once` from `std::sync` like this 👇🏼 so it doesn't call `::register()` everything the statusItem mounts

```rs
pub fn new(_workspace: &Workspace, cx: &mut AppContext) -> Self {
        static INIT: std::sync::Once = std::sync::Once::new();
        INIT.call_once(|| {
            ImageFileSizeUnitType::register(cx);
        });
 ```
 
 Lemme know if this is okay.